src/isIsogram.test.js

```diff
@@ -1,9 +1,41 @@
-'use strict';
+"use strict";
 
-describe('isIsogram', () => {
-  const { isIsogram } = require('./isIsogram');
+describe("isIsogram", () => {
+  const { isIsogram } = require("./isIsogram");
 
-  it(`should be declared`, () => {
+  it("should be declared", () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it("should throw error if argument not provided", () => {
+    try {
+      isIsogram();
+    } catch (error) {
+      expect(error).toBeInstanceOf(Error);
+    }
+  });
+
+  it("should return boolean value", () => {
+    expect(isIsogram("")).toBeInstanceOf(Boolean);
+  });
+
+  it("should return true if value equal empty string", () => {
+    expect(isIsogram("")).toBe(true);
+  });
```

Comment:
ur 2 checks look to see if a boolean(true) is returned when u have an empty string. If I were u, I would leave the second one
```suggestion
  it("should return true if value equal empty string", () => {
    expect(isIsogram("")).toBe(true);
  });
```